.trivyignore

```diff
@@ -1,8 +1,17 @@
-# openstack-objectstorage-k8s (https://git.launchpad.net/~canonical-sysadmins/wordpress/+git/openstack-objectstorage-k8s)
+CVE-2022-24775
 CVE-2022-29248
-CVE-2022-31042
+CVE-2022-31042 
```

Comment:
Nit: Could thsese CVEs be removed please?

Reply:
i know it says these CVEs are not present anymore and can be safely removed, but if i remove them the trivy scan fails - https://github.com/canonical/wordpress-k8s-operator/actions/runs/12081279416/job/33690383343